cg/meta/upload/nipt/nipt.py

```diff
@@ -0,0 +1,105 @@
+"""NIPT ftp upload API"""
+import datetime as dt
+import logging
+from pathlib import Path
+from typing import Optional
+
+from alchy import QueryModel
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import Pipeline
+from cg.exc import HousekeeperFileMissingError
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+from cg.utils import Process
+from housekeeper.store import Store
+
+LOG = logging.getLogger(__name__)
+
+
+class NiptUploadAPI:
+    """API for NIPT upload via ftp"""
+
+    RESULT_FILE_TAGS = ["nipt", "metrics"]
+
+    def __init__(self, config: CGConfig):
+        self.sftp_user: str = config.fluffy.sftp.user
+        self.sftp_password: str = config.fluffy.sftp.password
+        self.sftp_host: str = config.fluffy.sftp.host
+        self.root_dir = Path(config.housekeeper.root)
+        self.housekeeper_api: HousekeeperAPI = config.housekeeper_api
+        self.status_db: Store = config.status_db
+        self.process = Process(binary="lftp")
+        self.dry_run: bool = False
+
+    def set_dry_run(self, dry_run: bool) -> None:
+        """Set dry run"""
+
+        LOG.info("Set dry run to %s", dry_run)
+        self.dry_run = dry_run
+
+    def get_housekeeper_results_file(self, case_id: str, tags: Optional[list] = None) -> str:
+        """Get the result file for a NIPT analysis from Housekeeper"""
+
+        if not tags:
+            tags = self.RESULT_FILE_TAGS
+
+        hk_all_results_files = self.housekeeper_api.get_files(bundle=case_id, tags=tags)
```

Comment:
```suggestion
        hk_all_results_files: List[hk_models.File] = self.housekeeper_api.get_files(bundle=case_id, tags=tags)
```
Perhaps not correct but please type hint

Reply:
Turning this into `Iterable[hk_models.File]`, as suggested by Pycharm.

Reply:
suggested fix in d1e79180d6dac13d8d76a5a28085b6907a7ca4dc